src/actions/sources/newSources.js

```diff
@@ -44,15 +44,20 @@ function createOriginalSource(
  * @memberof actions/sources
  * @static
  */
-export function loadSourceMap(generatedSource: Source) {
+export function loadSourceMap(generatedSourceId: SourceId) {
   return async function({ dispatch, getState, sourceMaps }: ThunkArgs) {
-    let urls;
+    const generatedSource = getSource(getState(), generatedSourceId);
+    if (!generatedSource.sourceMapURL) {
```

Comment:
this new check keeps us from attempting to fetch source maps if the sources don't have a source map url.

Reply:
Makes sense.